4.8/runtime/windowsservercore-ltsc2016/Dockerfile

```diff
@@ -1,32 +1,33 @@
 # escape=`
+# soft newline =\
```

Comment:
@gesture1968, Can you point me to the documentation for this?  I have not seen this syntax before.